src/style.css

```diff
@@ -0,0 +1,125 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, Helvetica, sans-serif;
+  font-style: normal;
+}
+
+.label {
+  display: flex;
+  position: relative;
+  width: 100%;
+}
+
+.input {
+  display: flex;
+  width: 100%;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  color: #3d4e61;
+  outline: none;
+}
+
+.input_big {
+  height: 70px;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 1.38;
+  padding-left: 62px;
+}
+
+.icon {
+  position: absolute;
+  transform: translateY(-50%);
+  top: 50%;
+  display: flex;
+}
+
+.icon_big {
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.input_big:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.input_big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+  font-weight: bold;
+}
+
+.input_big::placeholder {
+  font-size: 16px;
```

Comment:
For different states of the same element, only those styles that will different are set, there is no need to duplicate them if they are set in the main class. Fix it everywhere
![image](https://user-images.githubusercontent.com/70009205/215257892-b4055bca-7285-4790-8b1f-378c86ef3901.png)

Reply:
[loralevitska]"Please deploy test_report link for review"